### PR TITLE
Improve api docs

### PR DESCRIPTION
**Pull request by michalkulakowski**
http://openvino-doc.iotg.sclab.intel.com/mkulakow/index.html

### CODE REVIEW

File: docs/clients_tfs.md [hunk 1]
```diff
@@ -167,24 +213,43 @@ When creating a Python-based client application, there are two packages on PyPi
         predict_response = prediction_service_stub.Predict(predict_request, 10.0)
         results = make_ndarray(predict_response.outputs["output_name"])
 
+.. tab:: curl
+
+    .. code-block:: sh  
+
+        curl http://localhost:9000/v1/models/my_model
```
Comment (dtrawins) [inline docs/clients_tfs.md:220]: this is not a complete command.

File: docs/clients_kfs.md [hunk 2]
```diff
@@ -23,55 +23,319 @@ When creating a Python-based client application, you can use Triton client libra
...
+        int main() {
+                        std::unique_ptr<tc::InferenceServerGrpcClient> client;
+                        tc::InferenceServerGrpcClient::Create(&client, "localhost:9000");
+
+                        bool serverLive = client->IsServerLive(&serverLive);
+
+                        bool serverReady = client->IsServerReady(&serverReady);
+
+                        bool modelReady = client->IsModelReady(&modelReady, "model_name", "model_version");
+                }
```
Comment (dtrawins) [inline docs/clients_kfs.md:70]: too many whitespaces

File: docs/clients_kfs.md [hunk 3]
```diff
@@ -82,17 +346,148 @@ When creating a Python-based client application, you can use Triton client libra
...
+
+        ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+
+        inferInputs := []*grpc_client.ModelInferRequest_InferInputTensor{
+            &grpc_client.ModelInferRequest_InferInputTensor{
+                Name:     "b",
+                Datatype: "FP32",
+                Shape:    []int64{1, 10},
+                Contents: &grpc_client.InferTensorContents{
+                    Fp32Contents: inputData,
```
Comment (dtrawins) [inline docs/clients_kfs.md:470]: what is inputData? maybe creating such a slice would help here

**Comment by dtrawins** [comment]
in ovms_docs_binary_input change also regular data to raw data

File: docs/clients_tfs.md [hunk 4]
```diff
@@ -33,16 +33,26 @@ When creating a Python-based client application, there are two packages on PyPi
...
 
 
+.. tab:: ovmsclient [REST]
+
+    .. code-block:: python
+
+        from ovmsclient import make_http_client
+
+        client = make_http_client("localhost:9000")
+        status = client.get_model_status(model_name="my_model", model_version="", 10.0)
```
Comment (mzegla) [inline docs/clients_tfs.md:53]: why this line any different than grpc version?

File: docs/clients_tfs.md [hunk 5]
```diff
@@ -33,16 +33,26 @@ When creating a Python-based client application, there are two packages on PyPi
...
         model_status = client.get_model_status(model_name="my_model")
 
 
+.. tab:: ovmsclient [REST]
+
+    .. code-block:: python
+
+        from ovmsclient import make_http_client
+
+        client = make_http_client("localhost:9000")
```
Comment (mzegla) [inline docs/clients_tfs.md:52]: I think it would be better to have a different port here

File: docs/clients_tfs.md [hunk 6]
```diff
@@ -66,24 +76,39 @@ When creating a Python-based client application, there are two packages on PyPi
...
+        model_metadata = client.get_model_metadata(model_name="my_model", model_version="", 10.0)
```
Comment (mzegla) [inline docs/clients_tfs.md:109]: Same as for status

File: docs/clients_tfs.md [hunk 7]
```diff
@@ -129,24 +154,45 @@ When creating a Python-based client application, there are two packages on PyPi
...
+
+        with open("img.jpeg", "rb") as f:
+            data = f.read()
+        inputs = {"input_name": data}    
+        results = client.predict(inputs=inputs, model_name="my_model", model_version="", 10.0)
```
Comment (mzegla) [inline docs/clients_tfs.md:194]: Same as for status. 
port could be different and additional parameters are not relevant here

File: docs/clients_tfs.md [hunk 8]
```diff
@@ -167,24 +213,43 @@ When creating a Python-based client application, there are two packages on PyPi
...
+
+        import numpy as np
+        from ovmsclient import make_http_client
+
+        client = make_http_client("localhost:9000")
+        model_metadata = client.get_model_metadata(model_name="my_model", model_version="", 10.0)
+
+        data = np.array([1.0, 2.0, ..., 1000.0])
+        inputs = {"input_name": data}
+        results = client.predict(inputs=inputs, model_name="my_model", model_version="", 10.0)
```
Comment (mzegla) [inline docs/clients_tfs.md:252]: Same as for status.
port could be different and additional parameters are not relevant here

File: docs/model_server_rest_api_kfs.md [hunk 9]
```diff
@@ -227,24 +227,47 @@ $request_output =
...
 {
 "model_name" : "my_model",
 "inputs" : [
    {
       "name" : "model_input",
       "shape" : [ 1 ],
-      "datatype" : "BYTES",
-      "parameters" : {
-         "binary_data_size" : "9472"
-      }
```
Comment (mzegla) [inline docs/model_server_rest_api_kfs.md:240]: Why is this part removed?

File: docs/clients_kfs.md [hunk 10]
```diff
@@ -326,10 +326,199 @@ When creating a Python-based client application, you can use Triton client libra
...
+            Inputs:       inferInputs,
+        }
+
+        modelInferResponse, err := client.ModelInfer(ctx, &modelInferRequest)
+
+.. tab:: curl    
+
+    .. code-block:: sh  
+
+        echo -n ‘{"inputs” : [{"name" : "0", "shape" : [1], "datatype" : "BYTES"}]}’ > request.json
```
Comment (dtrawins) [inline docs/clients_kfs.md:514]: wrong quote char

File: docs/clients_kfs.md [hunk 11]
```diff
@@ -326,10 +326,199 @@ When creating a Python-based client application, you can use Triton client libra
 
     .. code-block:: sh  
 
-        curl http://localhost:9000/v2/models/model_name
+        curl http://localhost:8000/v2/models/model_name
         
 @endsphinxdirective
 
+### Request Prediction on a Encoded Image
```
Comment (dtrawins) [inline docs/clients_kfs.md:333]: on an

**Review by mzegla — APPROVED** [review]

**Review by bstrzele — APPROVED** [review]
